spacksites/spack-env-templates/slurm/intel.yaml

```diff
@@ -40,54 +40,11 @@ spack:
         tcl: $env/modules
 
       tcl:
-        hide_implicits: true
-        all:
-          autoload: direct
-          # conflict with everything of same name
-          conflict:
-            - '{name}'
         exclude: ['glibc', 'gcc-runtime']
-        hash_length: 0
         intel-oneapi-mpi:
-          conflict:
-            - 'mpi'
           environment:
             set:
               SLURM_MPI_TYPE: 'pmi2'
```

Comment:
This is the only other bit that I'm not sure it should be system-specific or generic: the slurm MPI type for Intel oneAPI MPI.  It's both here and in Kathleen.

Reply:
Ok, will remove from this file then.